src/common/lib/fs.js

```diff
@@ -0,0 +1,5 @@
+import {promisifyAll} from 'bluebird';
+
+const fs = promisifyAll(require('fs'));
+
+export default fs;
```

Comment:
I am not sure if `fs` is a common/lib. Not sure what are the patterns, but there's no fs in RN & Web, so that's something that should go directly to server.